wsgidav/dc/pam_dc.py

```diff
@@ -37,6 +38,12 @@ def __init__(self, wsgidav_app, config):
         self.pam_service = dc_conf.get("service", "login")
         self.pam_encoding = dc_conf.get("encoding", "utf-8")
         self.pam_resetcreds = dc_conf.get("resetcreds", True)
+        self.allow_users = dc_conf.get("allow_users", "all")
+        assert self.allow_users in ("all", "current") or isinstance(self.allow_users, list), \
```

Comment:
asserts may be removed in a deployment build, so I think we should us `if / raise ValueError` instead, similar to `_check_config()`